common/app/views/support/GetClasses.scala

```diff
@@ -58,7 +58,7 @@ object GetClasses {
       containerDefinition.showLatestUpdate,
       containerDefinition.index == 0 && containerDefinition.customHeader.isEmpty,
       containerDefinition.displayName.isDefined,
-      containerDefinition.displayName == Some("headlines"),
+      containerDefinition.displayName.contains("headlines"),
```

Comment:
I'm not sure why, but I have been bitten by this about 3 times now and I **thought** I was on 2.11 but it then fails to compile and had to revert it.